src/main/java/com/cloudogu/smeagol/wiki/domain/Content.java

```diff
@@ -0,0 +1,55 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import com.google.common.base.Strings;
+
+import java.util.Objects;
+
+/**
+ * Content is a simple string value object.
+ */
+public class Content {
+
+    private final String value;
+
+    private Content(String value) {
+        this.value = value;
+    }
+
+    /**
+     * Returns the string representation of the content.
+     *
+     * @return string representation
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Creates a content from a string.
+     *
+     * @param value string representation of content
+     *
+     * @return repository name
+     */
+    public static Content valueOf(String value) {
+        return new Content(Strings.nullToEmpty(value));
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)